coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
+    "line": 30.05
+  }
+}
```

Comment:
👍🏻  Love that you were checking test coverage while developing this code. That helps TDD process run smoother.

Did you consider .gitignoring the resultant coverage files?